lib/atomic-management.js

```diff
@@ -44,32 +44,45 @@ export default {
       })
     );
 
-    atom.workspace.observeTextEditors(editor => {
-      editor.onDidSave(e => {
-      	// whenever users save, new configuration applies
+    // Register event handler to set project config when projects are
+    // added/removed from the window
+    this.subscriptions.add(
+      atom.project.onDidChangePaths(paths => this.setConfig())
```

Comment:
This should have the same `(!this.isEnabled)` as `onDidOpen` below

Reply:
In that case, maybe it would be cleaner to move the `(!this.isEnabled)` to inside the `setConfig` function?